### ISSUE TITLE
How to programmatically scroll to a specific item in the list?  - Question

### ISSUE THREAD

**Issue by golgistudio** [atlassian/react-beautiful-dnd#706]
Really terrific package!  Thank you!

## Bug or feature request?

Question

### Desired behavior

* When a user adds an item to a list, we would like them to be able to select where in the list to add the new item (e.g. after [item name]).
* Once the item has been added, the list should scroll to the newly added item.

**Question**: How can I programmatically scroll the list to a specific item in the list?

### What version of `React` are you using?

16.3.2

### What version of `react-beautiful-dnd` are you running?

7.1.3

### What browser are you using?

chrome

**Comment by alexreardon** [comment]
You control the creation of your list. You are welcome to scroll to any point in the list using `window.scrollTo`, `element.scrollIntoView` or something similar 👍

**Comment by mousikosk** [comment]
I'm surprised by how little I was able to find about this online. I thought that scrolling to the newly added element of a list would be a very common request. It is not trivial to me how one can achieve this. Should `scrollIntoView` be invoked on a ref to the `Draggable` component? Would it be possible to have a snippet of how one would make the simple list example https://react-beautiful-dnd.netlify.com/iframe.html?id=single-vertical-list--basic scroll to the end (where presumably a new element would have been added) using scrollIntoView?
